cmd/commands/start.go

```diff
@@ -80,23 +76,33 @@ func cmdStart(c *cli.Context) error {
 		basecoinApp.RegisterPlugin(p.newPlugin())
 	}
 
-	// If genesis file exists, set key-value options
-	genesisFile := path.Join(c.String("dir"), "genesis.json")
-	if _, err := os.Stat(genesisFile); err == nil {
-		err := basecoinApp.LoadGenesis(genesisFile)
-		if err != nil {
-			return errors.New(cmn.Fmt("%+v", err))
+	fmt.Println("CHAIN ID", basecoinApp.GetState().GetChainID())
```

Comment:
do we need to print CHAIN ID twice? If it is empty, the client will get 

"CHAIN ID"
"CHAIN ID test"

if not empty

"CHAIN ID test"
"CHAIN ID test"